src/configure

```diff
@@ -796,10 +796,9 @@ echo "CONFIGURE_VERSION := $CONFIGURE_VERSION" >> kaldi.mk
 echo "FSTROOT = $FSTROOT" >> kaldi.mk
 
 # Check installed OpenFst version and add C++11 flags if OpenFst >= 1.4
-OPENFST_VER=`grep 'PACKAGE_VERSION' $FSTROOT/Makefile | sed -e 's:.*= ::'`
+OPENFST_VER="${OPENFST_VER:-`grep 'PACKAGE_VERSION' $FSTROOT/Makefile | sed -e 's:.*= ::'`}"
 echo "OPENFST_VER = $OPENFST_VER" >> kaldi.mk
-OPENFST_VER_NUM=`echo $OPENFST_VER | sed 's/\./ /g' | xargs printf "%d%02d%02d"`
-if [ $OPENFST_VER_NUM -ge 10400 ]; then
+if [ `echo $OPENFST_VER | sed 's/\./0/g'` -ge 10400 ]; then
```

Comment:
> In src/configure
> <https://github.com/kaldi-asr/kaldi/pull/480#discussion_r51662717>:
>
> >  echo "OPENFST_VER = $OPENFST_VER" >> kaldi.mk
> > -OPENFST_VER_NUM=`echo $OPENFST_VER | sed 's/\./ /g' | xargs printf "%d%02d%02d"`
> > -if [ $OPENFST_VER_NUM -ge 10400 ]; then
> > +if [ `echo $OPENFST_VER | sed 's/\./0/g'` -ge 10400 ]; then
>
> One more command invocation, it's a kind of semi-automatic habit to
> simplify things. I just shortened it with the assumption that neither minor
> nor bugfix versions will exceed 99 (given the current release cycle of
> openfst this is really unlikely). I you wish I can revert that part, I
> really only care about the other part. Let me know.
>


Yes, revert that part.  I don't think it works even if the versions exceed
9, because then the ordering gets messed up.





> … <#1576033343_>
> On Tue, Feb 2, 2016 at 4:41 PM, Daniel Povey <notifications@github.com>
> wrote: In src/configure <#480 (comment)
> <https://github.com/kaldi-asr/kaldi/pull/480#discussion_r51662266>>: >
> echo "OPENFST_VER = $OPENFST_VER" >> kaldi.mk > -OPENFST_VER_NUM=`echo
> $OPENFST_VER | sed 's/\./ /g' | xargs printf "%d%02d%02d"` > -if [
> $OPENFST_VER_NUM -ge 10400 ]; then > +if [ `echo $OPENFST_VER | sed
> 's/\./0/g'` -ge 10400 ]; then What was the problem with the previous
> version? … <#1765803507_> On Tue, Feb 2, 2016 at 7:40 PM, Alexander
> Solovets < notifications@github.com > wrote: In src/configure <#480
> <https://github.com/kaldi-asr/kaldi/pull/480> (comment) <#480 (comment)
> <https://github.com/kaldi-asr/kaldi/pull/480#discussion_r51662181>>>: >
> echo "OPENFST_VER = $OPENFST_VER" >> kaldi.mk > -OPENFST_VER_NUM=`echo
> $OPENFST_VER | sed 's/\./ /g' | xargs printf "%d%02d%02d"` > -if [
> $OPENFST_VER_NUM -ge 10400 ]; then > +if [ `echo $OPENFST_VER | sed
> 's/\./0/g'` -ge 10400 ]; then The previous version wasn't perfect either,
> thus I guess the options are: 1. Use the previous version. 2. Implement
> proper comparison (imo, an overkill in this case). 3. Use string comparison
> (this is what the most of OSS do). What do you think? … <#1741592730_> On
> Tue, Feb 2, 2016 at 4:20 PM, Daniel Povey <notifications@github.com>
> wrote: In src/configure <#480
> <https://github.com/kaldi-asr/kaldi/pull/480> <#480
> <https://github.com/kaldi-asr/kaldi/pull/480>> (comment) <#480
> <https://github.com/kaldi-asr/kaldi/pull/480> (comment) <#480 (comment)
> <https://github.com/kaldi-asr/kaldi/pull/480#discussion_r51660266>>>>: >
> echo "OPENFST_VER = $OPENFST_VER" >> kaldi.mk > -OPENFST_VER_NUM=`echo
> $OPENFST_VER | sed 's/\./ /g' | xargs printf "%d%02d%02d"` > -if [
> $OPENFST_VER_NUM -ge 10400 ]; then > +if [ `echo $OPENFST_VER | sed
> 's/\./0/g'` -ge 10400 ]; then I don't think this change will work as
> intended if the minor revision or major revision exceeds 9. — Reply to this
> email directly or view it on GitHub <
> https://github.com/kaldi-asr/kaldi/pull/480/files#r51660266>. --
> Sincerely, Alexander — Reply to this email directly or view it on GitHub <
> https://github.com/kaldi-asr/kaldi/pull/480/files#r51662181>. — Reply to
> this email directly or view it on GitHub <
> https://github.com/kaldi-asr/kaldi/pull/480/files#r51662266>.
> -- Sincerely, Alexander
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/480/files#r51662717>.
>